packages/core/framework/src/http/middleware-file-loader.ts

```diff
@@ -90,6 +99,24 @@ export class MiddlewareFileLoader {
           })
         }
 
+        if (
+          "additionalDataValidator" in route &&
```

Comment:
suggestion(non-blocking): use isPresent util

Reply:
Doesn't narrow down the types 😔

![CleanShot 2025-04-16 at 15 03 50@2x](https://github.com/user-attachments/assets/2d260a46-a8fb-4fc9-b52e-617547e0fd2d)